### PR TITLE
Listview for others with rights

### PR DESCRIPTION
**Pull request by acn-dgopa**
Reused the classes from client delegation, removed unecessary css for listview, added css for dark blue collapse button

### CODE REVIEW

**Comment by matsgm** [comment]
Pull request needs to be documented in Releases vNext (talk to @magnusnn )

File: source/css/scss-common/objects/_collapse.scss [hunk 1]
```diff
@@ -496,27 +501,11 @@
...
-            margin-top: 0;
-          }
-        }
-      }
 
-      .hideText {
-        display: none;
-        }
-    }
+
```
Comment (magnusnn) [inline source/css/scss-common/objects/_collapse.scss]: gir ikke dette en lint-feil siden det er to spaces? `npm run lint`

Comment (acn-dgopa) [reply source/css/scss-common/objects/_collapse.scss]: fikset

**Comment by acn-dgopa** [comment]
> Pull request needs to be documented in Releases vNext (talk to @magnusnn )

Updated the release docs

**Pull request merged**